### ISSUE TITLE
java.lang.String cannot be cast to com.facebook.react.uimanager.AccessibilityDelegateUtil$AccessibilityRole

### ISSUE THREAD

**Issue by agustin107** [react-navigation/react-navigation#5122]
### Current Behavior

`App.js`
![image](https://user-images.githubusercontent.com/2467822/46894936-96f0cd00-ce4c-11e8-8211-0fc32645ca02.png)

`Home.js`
![image](https://user-images.githubusercontent.com/2467822/46894940-a2dc8f00-ce4c-11e8-9f5e-c54a48953baa.png)

### Expected Behavior

![image](https://user-images.githubusercontent.com/2467822/46894919-893b4780-ce4c-11e8-95ff-650ddd168a34.png)

### How to reproduce

Run `react-native run-android`

### Your Environment

| software         | version
| ---------------- | -------
| react-navigation | "^2.18.0"
| react-native     | "0.57.3"
| node             | "8.9.1"
| npm or yarn      |  "1.9.4"

**Comment by zakrza59** [comment]
@agustin107 it's because of button, same problem here

**Comment by WoodyGu** [comment]
I used button from another module, react-native-elements instead react-native and that worked.
import { Button } from 'react-native-elements';
